### PR TITLE
Create index for query performance

### PR DESCRIPTION
**Pull request by ekenecf**
Use EXPLAIN ANALYZE to analyze query-time 
Find a way to decrease the execution time

                                              **BEFORE IMPROVEMENT** 

- vet_id_index

- Explain_analyse3

- Explain_analyse

                                                 **AFTER IMPROVEMENT**

- email_analyze

- explain_analyse_answer1

- vet_id_index_answer

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -67,3 +67,17 @@ CREATE TABLE visits (
...
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+CREATE INDEX animal_id_desc
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+
+explain analyze SELECT * FROM visits where vet_id = 2;
+CREATE INDEX vet_index
+explain analyze SELECT * FROM visits where vet_id = 2;
+
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+CREATE INDEX email_desc ON owners(email DESC);
```
Comment (rotimiazeez) [inline schema.sql:82]: - Good job so far. Please note the you should include the screenshots in the PR description with the results of the  EXPLAIN ANALYZE command for each of the above queries (in total it should be 6 screenshots):

The first screenshot should show how much time the query takes BEFORE the improvements introduced,
- The execution time should be around 1 sec (1000 ms).
The second screenshot should show how much time the query takes AFTER the improvements introduced
- The execution time should be significantly lower than 1 sec (1000 ms).👍🏾

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @ekenecf 👏🏽, 
## Good job so far!👍🏾
### Summay 👏
- No linter errors ✔
- Good commit message 👊🏽
- correct gitflow followed😉
### There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (ekenecf) [reply schema.sql:82]: Hi @rotimiazeez, Kindly find the screenshots as requested in the project requirement

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ekenecf , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**